swagger.json

```diff
@@ -2676,7 +2676,20 @@
       "GoogleLoginRequest": {
         "summary": "An example of Google Login request",
         "value": {
-          "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9"
+          "id_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9"
+        }
+      },
+      "GoogleLoginResponse": {
```

Comment:
Please add this to LoginResponse and LoginUserResponse or replace those schemas usage with GoogleLoginResponse.

Ideally, everything should be consistent for both schema models and examples across all auth endpoints:
- login
- register
- google-login
- refresh-token